packages/dart_frog/lib/src/body_parsers/form_data.dart

```diff
@@ -37,3 +54,139 @@ bool _isFormUrlEncoded(ContentType? contentType) {
   if (contentType == null) return false;
   return contentType.mimeType == formUrlEncodedContentType.mimeType;
 }
+
+bool _isMultipartFormData(ContentType? contentType) {
+  if (contentType == null) return false;
+  return contentType.mimeType == multipartFormDataContentType.mimeType;
+}
+
+FormData _extractFormUrlEncodedFormData({required String body}) {
+  return FormData(Uri.splitQueryString(body), {});
+}
+
+Future<FormData> _extractMultipartFormData({
+  required Map<String, String> headers,
+  required Stream<List<int>> bytes,
+}) async {
+  final contentType = headers[HttpHeaders.contentTypeHeader];
```

Comment:
```suggestion
  final contentType = headers[HttpHeaders.contentTypeHeader]!;
```
nit but if we're confident it exists should we unwrap right away?